.github/ISSUE_TEMPLATE/new_rule.yaml

```
@@ -0,0 +1,106 @@
+name: New Rule
+description: Suggestions and ideas for new rules
+title: "[New Rule] Name of rule"
+labels: ["Rule: New", "Team: TRADE"]
+assignees: []
+projects: ["elastic/1268"]
+
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: "Provide a detailed description of the activity to be detected."
+      placeholder: "Detailed description..."
+
+  - type: dropdown
+    id: target_ruleset
+    attributes:
+      label: Target Ruleset
+      description: "Select the target rulset."
+      options:
+        - apm
+        - cross-platform
+        - aws
+        - aws_bedrock
+        - azure
+        - azure_openai
+        - beaconing
+        - cloud_defend
+        - cyberparkpas
+        - ded
+        - dga
+        - endpoint
+        - fim
+        - gcp
+        - github
+        - google_workspace
+        - kubernetes
+        - lmd
+        - o365
+        - okta
+        - problemchild
+        - linux
+        - macos
+        - ml
+        - network
+        - promotions
+        - threat_intel
+        - windows
+        - other
+
+  - type: dropdown
+    id: rule_type
+    attributes:
+      label: Target Rule Type
+      description: "Select the target type."
+      options:
+        - Custom (KQL or Lucene)
+        - Machine Learning
+        - Threshold
+        - Event Correlation (EQL)
+        - Indicator Match
+        - New Terms
+        - ES|QL
+
```

```suggestion
        - ES|QL
        - OSQuery
```

Should we have OSQuery be a rule type as well as hunt type?

---

Not in this case. This is specifically for the actual rule types we support.